.github/workflows/docs.yml

```diff
@@ -0,0 +1,21 @@
+name: Deploy docs
```

Comment:
Guessing this is blocked on #319 right now?

Reply:
Nope unintentional, don't know how this commit made it in here, will remove.